### PR TITLE
Test json data

### PR DESCRIPTION
**Pull request by viktoria-rybenchuk**

### CODE REVIEW

**Review by SafonovVladimir — APPROVED** [review]
Fine!

File: library_db_data.json [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    "pk": 3,
+    "fields": {
+      "borrow_date": "2022-05-15",
+      "expected_date": "2022-06-15",
+      "actual_date": "2022-06-10",
+      "book_id": [1, 3],
+      "user_id": 3
+    }
+  }
+]
```
Comment (Zenionoi) [inline library_db_data.json:193]: Add blank line

File: library_db_data.json [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  {
+    "model": "books.book",
+    "pk": 2,
+    "fields": {
+      "title": "Kafka on the Shore",
+      "author": [3],
+      "cover": "SOFT",
+      "image": "book_cover/kafka_on_the_shore.jpg",
+      "inventory": 12,
+      "daily_fee": 2.5
```
Comment (Zenionoi) [inline library_db_data.json]: We define daily_fee like IntegerField

**Review by Zenionoi — CHANGES_REQUESTED** [review]
I think we should to clearly define `daily_fee` properties

File: library_db_data.json (hunk 2 above)
Comment (TeliksPG) [reply library_db_data.json]: @Zenionoi it must be decimal

Comment (Zenionoi) [reply library_db_data.json]: Its true, but we have `daily_fee = models.IntegerField()` and nowhere work to transform dec to int

Comment (TeliksPG) [reply library_db_data.json]: so, we should change it)))

**Review by SafonovVladimir — APPROVED** [review]
GJ

**Review by TeliksPG — APPROVED** [review]
good

**Review by Zenionoi — APPROVED** [review]
GJ!

**Review by SafonovVladimir — APPROVED** [review]
GJ

**Pull request merged**